src/content/en/agencies/js/directory.js

```diff
@@ -0,0 +1,656 @@
+/**
+ * @fileoverview Creates web directory of agencies based on JSON data
+ * TODO: Eventually refactor this to be used as a 'library' for a variety of directories
+ */
+
+/**
+ * Initializing the entire directory
+ *
+ * @param {String} markup, defined in script in HTML file
+ * Example: "<div><% this.example_data %></div>"
+ *
+ * @param {String} requestUrl, path to relevant directory JSON file
+ */
+
+var initializeDirectory = function(markup, requestUrl, alphabeticalSortKey) {
```

Comment:
alphabeticalSortKey isn't documented

Reply:
let's use `function initializeDirectory() {}` for top level functions. Not function expressions.